### ISSUE TITLE
Questions about the name change

### ISSUE THREAD

**Issue by TheThirdOne** [andrescv/Jupiter#25]
I have been watching this project for a bit and was surprised to see a name change. Is there a specific reason for the change?

Is this project inspired by [MARS](http://courses.missouristate.edu/kenvollmar/mars/)? You mention SPIM as an inspiration, but with the name change to a planet it seems like MARS may also have been considered.

Also, in case you are not aware there are two  other risc-v simulators that were directly inspired from MARS, [venus](https://github.com/kvakil/venus) and [rars](https://github.com/thethirdone/rars) (my project).

**Comment by andrescv** [comment]
Hi sorry for the late response, when I started developing this project the main inspiration was SPIM because I was only planning to do a CLI in order to have a tool that could be used in a Compilers class in my University (2017-2018). But then a couple of professors ask me to do a GUI for other classes (this was inspired by Venus primary and by MARS).

I changed the name to Jupiter because of traditional reasons (Venus, MARS) and this planet represents in some sort the number five in contrast to RISC-V (the 5th gen of the original ISA designed in the 90's).

In the mean time, I realized that other great simulators have been developed and I would like to point to this others simulators (like yours) in my documentation in the last version when it comes out.
